activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -373,6 +388,23 @@ def group!(*args) # :nodoc:
     #
     #   User.order('name DESC, email')
     #   # SELECT "users".* FROM "users" ORDER BY name DESC, email
+    #
+    # This could be a source of SQL injection, so only strings composed of plain
+    # column names and simple function(column_name) expressions with optional
+    # ASC/DESC modifiers are allowed.
+    #
+    # === arel
```

Comment:
Do we want to encourage people to use Arel in the docs since it's (at least currently) a private API?

Reply:
`order`'s error message encourages using Arel if anything more complicated is passed.

I'll paste the error message before I mark this resolved.

Reply:
Error message recommending Arel:
```
ActiveRecord::UnknownAttributeReference: Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "payload->>'kind'".This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql().
```